docs/usage/items/datetime.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: default
+title: DateTime
+nav_order: 6
+has_children: false
+parent: Items
+grand_parent: Usage
+---
+
+# DateTime Item
+
+DateTime items are extended with a lot of methods that make working with them easy. Most ot the methods
+defined by the [ruby Time class](https://ruby-doc.org/core-2.5.8/Time.html) are available, and some of 
+them are extended or adapted with OpenHAB specific functionality.
+
+##### Examples
+
+DateTime Items can be updated and commanded with ruby Time objects
+
+```ruby
+Example_DateTimeItem << Time.now
+```
+
+Math operations (+ and -) are available to make calculations with time in a few different ways
+
+```ruby
+Example_DateTimeItem + 600 # Number of seconds
+Example_DateTimeItem - '01:15' # Subtracts 1h 15 min
+Example_DateTimeItem + 2.hours # Use the helper library's duration methods
+
+Example_DateTimeItem - Example_DateTimeItem2 # Calculate the time difference, in seconds
+Example_DateTimeItem - '2021-01-01 15:40' # Calculates time difference
+```
+
+Comparisons between different time objects can be performed
+
+```ruby
+Example_DateTimeItem == Example_DateTimeItem2 # Equality, works across time zones
+Example_DateTimeItem > '2021-01-31' # After midnight jan 31st 2021
```

Comment:
Does this work across different locales?

Reply:
It uses the OH DateTimeType parser, which by uses the system default timezone. But it might be a good idea to move some of the logic to ruby so it can be set to the same timezone as the first operand.

Reply:
I rewrote the logic for this so it uses the timezone from the first operand. Should work more as expected now.